sdk/eventhubs/azure-messaging-eventhubs/inc/azure/messaging/eventhubs/consumer_client.hpp

```diff
@@ -16,6 +15,10 @@
 #include <azure/core/http/policies/policy.hpp>
 #include <azure/core/internal/diagnostics/log.hpp>
 namespace Azure { namespace Messaging { namespace EventHubs {
+
+  /// @brief The default consumer group name.
+  constexpr const char* DefaultConsumerGroup = "$Default";
```

Comment:
Do we typically have public constant string values like these? I wasn't sure if we have guidelines for these (I know we try to scope them in namespaces, but not sure beyond that).

Reply:
This is scoped to a namespace (it's in the EventHubs namespace). 

Each EventHubs consumer is scoped to a specific consumer group. There is a default consumer group named "$Default" and I need to have a manifest constant be available for it. 

I'm open to alternate suggestions about how I can express the construct of `DefaultConsumerGroup`.